okcurl/src/main/kotlin/okhttp3/curl/Main.kt

```diff
@@ -39,21 +40,30 @@ import okhttp3.internal.platform.Platform
 import okhttp3.logging.HttpLoggingInterceptor
 import okhttp3.logging.LoggingEventListener
 
-class Main : CliktCommand(name = NAME, help = "A curl for the next-generation web.") {
+class Main : CliktCommand(name = NAME) {
+  override fun help(context: Context): String {
+    return "A curl for the next-generation web."
+  }
+
   val method: String? by option("-X", "--request", help = "Specify request command to use")
 
   val data: String? by option("-d", "--data", help = "HTTP POST data")
 
   val headers: List<String>? by option("-H", "--header", help = "Custom header to pass to server").multiple()
 
-  val userAgent: String by option("-A", "--user-agent", help = "User-Agent to send to server").default(NAME + "/" + versionString())
+  val userAgent: String by option(
+    "-A",
+    "--user-agent",
+    help = "User-Agent to send to server",
```

Comment:
Help can be passed using `.help()` function to prevent wrapping on arguments and instead wrap on function calls.